About.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="about.css">
+    <title>About - Darko</title>
+</head>
+<body>
+    
+    <div class="socials">
+        <ul class="social-m">
+            <li><a href="#" class="nono facebook-to"><i class="fab fa-facebook-f"></i></a></li>
+            <li><a href="#" class="nono twitter-to"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#" class="nono"><p class="fab">English</p></a></li>
+            <li><a href="#" class="nono"><p class="fab">Log out</p></a></li>
+        </ul>
+    </div>
+    <nav class="nav-bar">
+        <span class="btn humberger-btn"><i class="fas fa-bars" onclick="changeIcon(this)"></i></span>
+        <label class="logo"><a href="index.html" class="refr"><img src="IMAGES/darko-logo.png" alt="logo" class="logo-desk"></a></label>
+        <ul class="menu">
+            <li class="menu-choice home"><a href="index.html" class="link-desk">Home</a></li>
+            <li class="menu-choice about"><a href="About.html" class="link-desk">About</a></li>
+            <li class="menu-choice blog"><a href="" class="link-desk">Blog</a></li>
+            <li class="menu-choice news"><a href="" class="link-desk">News</a></li>
+            <li class="menu-choice sponsor"><a href="" class="link-desk">Sponsor</a></li>
+            <li class="menu-choice call" id="contact"><a href="" class="link-desk link-desko">Call Us</a></li>
+        </ul>
+    </nav>
+    <ul class="options">
+        <li class="choice"><a href="index.html" class="go-to-page">Home</a></li>
+        <li class="choice "><a href="About.html" class="go-to-page">About</a></li>
+        <li class="choice"><a href="#" class="go-to-page">Blog</a></li>
+        <li class="choice"><a href="#" class="go-to-page">Sponsor</a></li>
+        <li class="choice"><a href="#" class="go-to-page">News</a></li>
+        <li class="choice"><a href="#" class="go-to-page">Contact</a></li>
```

Comment:
- [ ] [OPTIONAL] Good job implementing the suggestion by the previous reviewer, but I strongly suggest you give the mobile menu a `position:fixed` and a higher `z-index` so that you can scroll the page when the mobile menu appears